R/html-deps.R

```diff
@@ -24,6 +24,9 @@ createWebDependency <- function(dependency, scrubFile = TRUE) {
   if (!inherits(dependency, "html_dependency"))
     stop("Unexpected non-html_dependency type")
 
+  # Resolve tagFunction()
+  dependency <- htmltools::resolveDependencies(list(dependency))[[1]]
```

Comment:
I think we agreed to remove this.

Reply:
Yep, this PR is rebased off of #3062 and I just haven't updated this one yet